Backends/System/Linux/Sources/Kore/System.cpp

```diff
@@ -110,6 +110,16 @@ namespace {
 
 int createWindow(const char* title, int x, int y, int width, int height, Kore::WindowMode windowMode, int targetDisplay, int depthBufferBits,
                  int stencilBufferBits) {
+
+	int nameLength = strlen(Kore::System::name());
+	char strName[nameLength+1];
+	strcpy(strName, Kore::System::name());
+	char strClass[] = "_KoreApplication";
+	char strNameClass[nameLength+17];
```

Comment:
This is not valid in a `cpp` file.  This is valid C code in the most updated standard, but is not part of the C++ standard, and as such it 'might' work in 'some' cases with 'certain' flags.

Also, shouldn't this be configurable by the configuration?